static/js/modules/legal-search.js

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+var $ = require('jquery');
+var URI = require('urijs');
+var _ = require('underscore');
+var urls = require('fec-style/js/urls');
+var FilterPanel = require('fec-style/js/filter-panel').FilterPanel;
+var filterTags = require('fec-style/js/filter-tags');
+
+var ao_template = require('../../templates/legal-search/ao.hbs');
+
+// Just lifted this from calendar-helpers.js
+function getUrl(path, params) {
```

Comment:
Are we using some kind of implicit namespacing system which prevents these from becoming globals?